stripe_terminal/ios/Classes/StripeTerminalApi.swift

```diff
@@ -1,9 +1,17 @@
 import Flutter
 
-struct PlatformError: Error {
+class PlatformError: Error {
     let code: String
     let message: String?
     let details: String?
+
+    init(
+        _ code: String,
+        _ message: String? = nil,
+        _ details: String? = nil
+    ) {
+        
+    }
```

Comment:
to avoid to specify the init with nil, just keep the struct and add the default value as nil